utils_linux.go

```diff
@@ -171,6 +171,9 @@ func createPidFile(path string, process *libcontainer.Process) error {
 	if err != nil {
 		return err
 	}
+	if _, err := os.Stat(path); err == nil {
+		return fmt.Errorf("file %s already exists", path)
```

Comment:
As the host files are getting replaced, adding documentation will not be suffice here.
we can ensure runc not override not existing files.